torchvision/io/_video_opt.py

```diff
@@ -14,6 +14,13 @@
 except (ImportError, OSError):
     _HAS_VIDEO_OPT = False
 
+try:
+    #_load_library("Decoder")
```

Comment:
This doesn't seem to work, hence hardcoded the path below.

Reply:
Discussed offline, we just need to `cd ..` to avoid picking `torchvision` from the current folder (which doesn't contain the `.so` files).

In general, I would recommend installing locally with `python setup.py develop`, so that the `.so` files are available and you won't face those problems in the future